cernopendata/modules/fixtures/data/docs/cms-guide-luminosity-calculation/cms-guide-luminosity-calculation.md

```diff
@@ -0,0 +1,275 @@
+# How to calculate luminosity for CMS open data
+
+## What is this?
+
+This document provides instruction for how to calculate the luminosity information
+for CMS in the open data environment. This is done using `brilcalc`, which is a command line tool from the CMS Beam Radiation Instrumentation and Luminosity (BRIL) group.
+
+Before you continue, consider that some of the information that you wish to calculate is already available for [2010](http://opendata-dev.web.cern.ch/record/1050), [2011](http://opendata-dev.web.cern.ch/record/1051), and [2012](http://opendata-dev.web.cern.ch/record/1052) data. The information available includes:
+
+* the integrated luminosity for validated runs and luminosity sections
+* the trigger prescales by run and luminosity section
+* the luminosity by luminosity section for the list of validated runs and luminosity sections
+
+## Install the CMS environment
+
+There are two options for installing the CMS environment, using a Virtual
+Machine (VM) or using a Docker container.
+
+### Virtual machine
+
+Follow the instructions [here](http://opendata.cern.ch/docs/cms-virtual-machine-2011) to install the CMS VM and then the instructions for [getting started](http://opendata.cern.ch/docs/cms-getting-started-2011).
+
+Briefly, to get started once the VM is installed, log in to the VM with the user name `cms-opendata`. Once logged in, click on the `CMS Shell` icon on the desktop to open the terminal.
+
+At the terminal prompt, run the command
+
+`
+$ cmsrel CMSSW_5_3_32
+`
+
+and then change directory
+
+`
+$ cd CMSSW_5_3_32/src
+`
+
+ and then finally intialize the CMS environment
+
+ `
+$cmsenv
+ `
+
+Now we are ready to [install the `brilcalc` package.](#install-brilcalc)
+
+### Docker container
+
+Following the instructions [here](http://opendata.cern.ch/docs/cms-guide-docker) we download and install an image
+and run a Docker container using the following command:
+
+`docker run --name opendata-2011 -it cmsopendata/cmssw_5_3_32 /bin/bash`
+
+When completed successfully we should be at the command prompt:
+
+`cmsusr@a17b2a79d067 ~/CMSSW_5_3_32/src $`
+
+Now we are ready to [install the `brilcalc` package.](#install-brilcalc)
+
+## Install brilcalc
+
+### cvmfs (preferred)
+If you have access to `cvmfs`:
+
+1. Set your `PATH`:
+
+  `export PATH=$HOME/.local/bin:/cvmfs/cms-bril.cern.ch/brilconda/bin:$PATH`
```

Comment:
This is not rendering properly.  Markdown starts a new list (from one, despite the 2) after the code line.  Google says an indentation of 4 spaces is needed for the code .... I will try to test.  But this and the next list are not numbering correctly.

Reply:
It looks fine to me on [Github in the browser](https://github.com/tpmccauley/open-data.cern.ch/blob/75ed26e0ec01b85de5db2779a85576c00f0286fc/cernopendata/modules/fixtures/data/docs/cms-guide-luminosity-calculation/cms-guide-luminosity-calculation.md#install-brilcalc)

![image](https://user-images.githubusercontent.com/496503/91148202-39b53c00-e6b1-11ea-9300-c0c2aa2e9518.png)

as well as using md preview on atom. What are you using?

Reply:
I tested this following the [open portal developing instructions](https://github.com/cernopendata/opendata.cern.ch/blob/master/DEVELOPING.rst) to get the live site locally.

Reply:
Ah ha. I installed python markdown via pip and generated an html file. I can reproduce the error. Will correct it.

Reply:
My install of the portal with docker failed at xrootd install step but managed to (I think) fix issues with latest commit

Reply:
Funny.  I had the same problem a few minutes ago.  But now it seems ok, maybe it was something with the yum repos.

Reply:
Yep, it looks good now.  I was able to lauch the live site again and it renders fine.